### PR TITLE
chore(swap): handle deposit received event name change [WEB-1010]

### PR DESCRIPTION
**Pull request by GabrielBuragev**

### CODE REVIEW

File: packages/swap/src/event-handlers/index.ts [hunk 1]
```diff
@@ -55,14 +57,16 @@ export const events = {
     EgressScheduled: 'EthereumIngressEgress.EgressScheduled',
     BatchBroadcastRequested: 'EthereumIngressEgress.BatchBroadcastRequested',
     CcmBroadcastRequested: 'EthereumIngressEgress.CcmBroadcastRequested',
-    DepositReceived: 'EthereumIngressEgress.DepositReceived',
+    DepositReceived: 'BitcoinIngressEgress.DepositReceived', // Renamed to DepositFinalised since spec 140
+    DepositFinalised: 'BitcoinIngressEgress.DepositFinalised',
```
Comment (niklasnatter) [inline packages/swap/src/event-handlers/index.ts:61]: arent you adding the `DepositFinalised` in the `spec: 140` block below too?

Comment (niklasnatter) [reply packages/swap/src/event-handlers/index.ts:61]: oh im stupid, can forget about this 🙈

File: packages/swap/src/event-handlers/index.ts [hunk 2]
```diff
@@ -193,6 +198,13 @@ const handlers = [
       ]),
     ],
   },
+  {
+    spec: 140,
+    handlers: Object.values(Chains).map((chain) => ({
+      name: events[`${chain}IngressEgress`].DepositFinalised,
+      handler: depositFinalised,
```
Comment (zoheb391) [inline packages/swap/src/event-handlers/index.ts:205]: can we add tests for this new event handler?

Comment (GabrielBuragev) [reply packages/swap/src/event-handlers/index.ts:205]: Yeah, i wanted to do that in a follow up PR so we unbrick the deposits on bspin.

Comment (zoheb391) [reply packages/swap/src/event-handlers/index.ts:205]: isn't it just a copy paste of the other file?

Comment (GabrielBuragev) [reply packages/swap/src/event-handlers/index.ts:205]: Oh, i thought there is no test for `depositReceived` as i couldn't find it in the `v120` folder. In that case i'll copy and adjust that test here.

Comment (GabrielBuragev) [reply packages/swap/src/event-handlers/index.ts:205]: Ready sir

**Review by zoheb391 — APPROVED** [review]

**Pull request merged**